yurt/yurt_core/utils.py

```diff
@@ -253,3 +254,130 @@ def register_values_in_vault(vagrantfile_path, vault_path, save_dict, quoted=Fal
         print('Cannot connect to vault: Connection Error')
 
 
+def go_over_questions(question_dict, current_value_dict, result=None, handle_defaults=None):
```

Comment:
It would make sense to do so for the non-YAML/file operating util functions, I think. Otherwise, we'd just be writing tests for the PyYAML library and built-in open methods